actioncable/app/assets/javascripts/action_cable/connection.coffee

```diff
@@ -74,10 +83,12 @@ class ActionCable.Connection
 
   events:
     message: (event) ->
+      return if not @isActive
```

Comment:
This isn't right. `@isActive` is a function, not a property.

I *think* we want to check that the WebSocket is using a supported protocol here so messages aren't received during the brief window after successfully connecting using an unsupported protocol.
```coffee
message: (event) ->
  return unless @isSupportedProtocol()
  ...
```